project/sming/provision.sh

```diff
@@ -23,7 +23,8 @@ fi
 HAS_SMING_HOME=`cat $PROFILE_CONF | grep SMING_HOME || :`
 if [ -z "$HAS_SMING_HOME" ]; then
   sudo echo "export ESP_HOME=/opt/Espressif" >> $PROFILE_CONF
-  sudo echo "export SMING_HOME=/home/vagrant/dev/Sming/Sming/" >> $PROFILE_CONF 
+  sudo echo "export SMING_HOME=/home/vagrant/dev/Sming/Sming/" >> 
+$PROFILE_CONF 
```

Comment:
Please, make sure the command is not split into two lines. $PROFILE_CONF should stay in the same line.